styles/colors-silicon-night.less

```diff
@@ -11,8 +11,8 @@
 @agate-bg-color: black;
 @agate-bg-image: none;
 @agate-bg-image2: none;
-@agate-text-color: @agate-foreground;
-@agate-title-text-color: @agate-foreground;
+@agate-text-color: @agate-white;
```

Comment:
Please leave the value to @agate-foreground + update the value of that variable for silicon-night.

// Abstracted Colors
@agate-background: black;
@agate-foreground: white;